kafka-connect-s3/src/main/java/io/confluent/connect/s3/hooks/BlockingKafkaPostCommitHook.java

```diff
@@ -49,20 +49,8 @@ public class BlockingKafkaPostCommitHook implements PostCommitHook {
 
   @Override
   public void init(S3SinkConnectorConfig config) {
-    @SuppressWarnings("unchecked")
-    Class<? extends Partitioner<?>> partitionerClass =
-            (Class<? extends Partitioner<?>>) config.getClass(
-                    PartitionerConfig.PARTITIONER_CLASS_CONFIG);
-
-    if (partitionerClass == null || !partitionerClass.getName().equals(
-            "io.logz.kafka.connect.FieldAndTimeBasedPartitioner")) {
-      throw new IllegalArgumentException("This post commit hook can be used ony with"
-              + " io.logz.kafka.connect.FieldAndTimeBasedPartitioner partitioner");
-    }
```

Comment:
This code was already removed from the current fat jar but we didn't sync master branch with the change, so removing it in order to sync it with the change.